Move everything about submitting to evaluation queue doc (#642)
Sage-Bionetworks/synapseDocs: articles/evaluation_queues.md

Any Synapse Entity may be submitted to an Evaluation Queue.

In the R and Python examples, you need to know the ID of the evaluation queue which you you are submitting to.  This ID is usually provided to you by administrators of the queue. 

> The submission function takes **two optional parameters**: `name` and `team`.  Name can be provided to serve as a custom name of the submission. If a name is not provided, the name of the entity being submitted will be used. Teams can optionally be provided to give credit to members of the team that contributed to the submission.

Author: The `name` here corresponds to the `name` of a submission.  So you can specify a name of a submission, but if it isn't specified, then the `name` of a Synapse Entity will be specified in the place of it.  So as an example if you have a Synapse File named `testfile.txt`.  If you don't specify the submission name, the submission will be named `testfile.txt`.  The `submission name` is used for participants to identify what they are submitting.

> The submission function takes **two optional parameters**: `name` and `team`.  Name can be provided to serve as a custom name of the submission. If a name is not provided, the name of the entity being submitted will be used. Teams can optionally be provided to give credit to members of the team that contributed to the submission.

Reviewer: I think it is appropriate to include an example like this.

    team = "My Team Name") # Optional, can also pass a Team object or id
```

##### Web

> Navigate to an uploaded file in Synapse and click on `Tools` on the upper right hand corner.

Reviewer: ```suggestion
Navigate to a file in Synapse and click on **Tools** in the upper right-hand corner.
```

> Select `Submit To Challenge`.  After doing so, pick the challenge you want to submit to, and follow the steps to complete your submission.

Reviewer: ```suggestion
Select **Submit To Challenge** to pick the challenge for your submission. Follow the provided steps to complete your submission.
```
